app/assets/javascripts/discourse/tests/acceptance/user-bookmark-bulk-actions-test.js

```diff
@@ -0,0 +1,77 @@
+import { click, visit } from "@ember/test-helpers";
+import { test } from "qunit";
+import {
+  acceptance,
+  count,
+  exists,
+  queryAll,
+} from "discourse/tests/helpers/qunit-helpers";
+import selectKit from "discourse/tests/helpers/select-kit-helper";
+import I18n from "discourse-i18n";
+
+acceptance("Bookmark - Bulk Actions", function (needs) {
+  needs.user();
+
+  test("bulk select - modal", async function (assert) {
+    await visit("/u/eviltrout/activity/bookmarks");
+    assert.ok(exists("button.bulk-select"));
+
+    await click("button.bulk-select");
+
+    await click(queryAll("input.bulk-select")[0]);
+    await click(queryAll("input.bulk-select")[1]);
+
+    const dropdown = selectKit(".select-kit.bulk-select-bookmarks-dropdown");
+    await dropdown.expand();
+
+    const options = dropdown.displayedContent();
+
+    assert.strictEqual(
+      options[0].name,
+      I18n.t("js.bookmark_bulk_actions.clear_reminders.name"),
+      "it shows an option to clear reminders"
+    );
+
+    assert.strictEqual(
+      options[1].name,
+      I18n.t("js.bookmark_bulk_actions.delete_bookmarks.name"),
+      "it shows an option to delete bookmarks"
+    );
```

Comment:
I don't think checking the presence of the options is necessary, since we're selecting each option just below and therefore implicitly ensuring their presence.